From 64b7731d28f548ebf8cca83da05df45fb172d720 Mon Sep 17 00:00:00 2001
From: Saravanakumar Duraisamy <quic_saradura@quicinc.com>
Date: Thu, 7 Sep 2023 16:58:30 +0530
Subject: [PATCH] wifi: mac80211: Remove unwanted semicolon at TRACE_EVENT

compilation error due to unwanted semi-colon at TRACE_EVENT
declaration.Remove unwanted semicolon.

depends on: 199-001-mac80211-add-nss-support.patch

Signed-off-by: Saravanakumar Duraisamy <quic_saradura@quicinc.com>
---
 net/mac80211/trace.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/mac80211/trace.h b/net/mac80211/trace.h
index 297bb01..1a2a66a 100644
--- a/net/mac80211/trace.h
+++ b/net/mac80211/trace.h
@@ -401,7 +401,7 @@ TRACE_EVENT(drv_nss_bss_info_changed,
 		LOCAL_ENTRY
 		VIF_ENTRY
 		__field(u32, changed)
-		__field(bool, nss_ap_isolate);
+		__field(bool, nss_ap_isolate)
 	),
 
 	TP_fast_assign(
-- 
2.17.1

